mask/eolearn/mask/cloud_mask.py

```diff
@@ -53,7 +53,7 @@ class AddCloudMaskTask(EOTask):
     """
     def __init__(self, classifier, data_feature, cm_size_x=None, cm_size_y=None, cmask_feature='CLM',
                  cprobs_feature=None, instance_id=None, data_collection=None,
-                 image_format=MimeType.TIFF, model_evalscript=MODEL_EVALSCRIPT, data_source=None):
+                 image_format=MimeType.TIFF, model_evalscript=None, data_source=None):
```

Comment:
We should remove these parameters (`image_format`, `model_evalscript`, `data_source`, `instance_id`, `data_collection`)? Either already available data (in eopatch) or (official) evalscript from the s2cloudless to download the data should be used. 

_If not now, when s2cloudless is updated..._

Reply:
Yes, in the next release we will significantly refactor this.